src/Strategy/MobileStrategy.php

```diff
@@ -40,10 +40,10 @@ class MobileStrategy implements StrategyInterface
      */
     public function __construct(\Mobile_Detect $mobileDetect = null)
     {
-        if (is_null($mobileDetect)) {
-            $this->MobileDetect = new \Mobile_Detect;
-        } else {
+        if ($mobileDetect !== null) {
```

Comment:
During refactoring we can replace whole if block with:
`$this->MobileDetect = $mobileDetect ?: new \Mobile_Detect;
`